layouts/default.vue

```diff
@@ -66,8 +66,19 @@ export default {
       }
     },
   },
+  watch: {
+    $route: {
+      handler(routeObj) {
+        if (routeObj.params.datasetIds === undefined) {
```

Comment:
Question: 
Isn't it saver to use:  if (!routeObj.params.datasetIds) ?

Reply:
Although `!routeObj.params.datasetIds` would probably work just as well, I opted for a more fixed notation because it is the layout which takes responsibility here. However, ideally, the layout does not need to know about the params and their keys. Therefor, `datasetIds` can become anything (`0` for instance). By being more specific, the change of a false positive is reduced. As an added bonus, it is more readable than the `!`.